Bracket.java

```diff
@@ -0,0 +1,83 @@
+import java.util.ArrayList;
+
+/**
+ * Bracket Class
+ * Created by Matt and Dan on 5/1/2017.
+ */
+public class Bracket {
+    //Attributes
+    ArrayList<String> bracket;
+    String name;
+    static final int EAST_BRACKET = 63;

```

Comment:
These are pointing at leaves in the starting round of 64 - it might make more semantic sense to point at the ultimate parent of these teams (indices 3-6).